products/bridge/smart-contracts/contracts/periphery/TokenManager.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity ^0.8.20;
+
+import {Relayer, CallMetadata} from "contracts/core/Relayer.sol";
+
+struct AcceptArgs {
+    address token;
+    address recipient;
+    uint amount;
+}
+
+struct RemoteToken {
+    address token;
+    address tokenManager;
+    uint chainId;
+}
+
+abstract contract TokenManager {
+    Relayer gateway;
+    // ChainId => token => remoteToken
```

Comment:
The comment is a bit misleading. The mapping suggests that the correct comment should be `token => remoteToken` only as the `chainId` is not a key but a property of the value stored in the mapping

Reply:
Ah yes, I had forgotten to remove it.